src/stores/uiOrderStore.ts

```diff
@@ -212,6 +227,64 @@ class UiOrderStore extends BaseStore {
           precisionFloor(+convertedBalance, this.quantityAccuracy);
   };
 
+  setMarketTotal = (
+    operationVolume?: string | number,
+    operationType?: Side,
+    debounce?: boolean
+  ) => {
+    if (
+      (!operationVolume && !operationType && !this.marketTotal.canBeUpdated) ||
+      (operationVolume &&
+        operationType &&
+        !this.marketTotal.canBeUpdated &&
+        debounce)
+    ) {
+      return;
+    } else if ((!operationVolume && !operationType) || debounce) {
+      this.marketTotal.canBeUpdated = false;
+      setTimeout(
```

Comment:
Why do you set value only after some time? Not sure that timeout should be here

Reply:
We should use temporary delay here - it was a part of this task (to avoid too frequent recounting of market total price)

Reply:
Then set a value in timeout, current solution looks like a hack to fix async issues